Frontend/src/components/TaskList.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import CsrfToken from './CsrfToken';
+
+export interface ITask {
+  id: number;
+  title: string;
+  status: boolean;
+}
+
+export async function getTasks(): Promise<ITask[]> {
+  const resp = await axios.get('list_tasks');
+  return resp.data.tasks;
+}
+
+export default function TaskList() {
```

Comment:
Great job with this component! It's clean code that only does what it's supposed to do. Meaning there's no additional code partially implemented for future use, which is good!